pkg/scheduler/eventhandlers.go

```diff
@@ -43,9 +43,10 @@ import (
 )
 
 func (sched *Scheduler) onStorageClassAdd(obj interface{}) {
+	logger := sched.logger
 	sc, ok := obj.(*storagev1.StorageClass)
 	if !ok {
-		klog.ErrorS(nil, "Cannot convert to *storagev1.StorageClass", "obj", obj)
+		logger.Error(nil, "Cannot convert to *storagev1.StorageClass", "obj", obj)
```

Comment:
Also think klog is enough. Then maybe we can remove logger from scheduler.

Reply:
Can you explain more about this? Why unit/integration tests will impact this, the logger was inherited from scheduler, it has no context of the tests. 

And since we added no names or values here, so the log output should be same.